From 37d9855be164c9d11c46ebd4ef5a9c41657b26cb Mon Sep 17 00:00:00 2001
From: Alex Zhao <zzc@rock-chips.com>
Date: Wed, 4 Jul 2018 17:26:33 +0800
Subject: [PATCH] net: wireless: rockchip_wlan: rtl8723bs: enable
 CONFIG_RESUME_IN_WORKQUEUE

Wifi does not take up system wake-up time

Change-Id: Ib226479f6146431d4ce2292f60a03e177d5874d1
Signed-off-by: Alex Zhao <zzc@rock-chips.com>
---
 drivers/net/wireless/rockchip_wlan/rtl8723bs/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/wireless/rockchip_wlan/rtl8723bs/Makefile b/drivers/net/wireless/rockchip_wlan/rtl8723bs/Makefile
index 46880dd7954a..0ff707fd37eb 100644
--- a/drivers/net/wireless/rockchip_wlan/rtl8723bs/Makefile
+++ b/drivers/net/wireless/rockchip_wlan/rtl8723bs/Makefile
@@ -1349,6 +1349,7 @@ EXTRA_CFLAGS += -DCONFIG_CONCURRENT_MODE
 # default setting for Power control
 EXTRA_CFLAGS += -DRTW_ENABLE_WIFI_CONTROL_FUNC
 EXTRA_CFLAGS += -DRTW_SUPPORT_PLATFORM_SHUTDOWN
+EXTRA_CFLAGS += -DCONFIG_RESUME_IN_WORKQUEUE
 # default setting for Special function
 ARCH := arm
 CROSS_COMPILE := /home/android_sdk/Rockchip/Rk3188/prebuilts/gcc/linux-x86/arm/arm-eabi-4.6/bin/arm-eabi-
-- 
2.35.3

